certified-connectors/PROS Sales Copilot/apiProperties.json

```diff
@@ -0,0 +1,35 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "{{CLIENT_ID}}",
+          "scopes": [],
+          "redirectMode": "GlobalPerConnector",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect/{{CONNECTOR_ID}}",
```

Comment:
Please remove redirectUrl, this is not required to mention here.  Team will guide you to mention this in your client app during certification process.

Reply:
Thanks.